cluster-autoscaler/cloudprovider/aws/CA_with_AWS_IAM_OIDC.md

```diff
@@ -0,0 +1,171 @@
+#### The following is an example to make use of the AWS IAM OIDC with the Cluster Autoscaler in an EKS cluster. 
+
+
+#### Prerequisites 
+
+  - An Active EKS cluster (1.14 preferred since it is the latest) against which the user is able to run kubectl commands. 
+  - Cluster must consist of at least one worker node ASG. 
+
+A) Create an IAM OIDC identity provider for your cluster with the AWS Management Console using the [documentation] . 
+
+B) Create a test [IAM policy] for your service accounts.
+
+```sh
+{
```

Comment:
instead of an example policy, can we add policies cluster-autoscaler need here directly??

Reply:
The example policy given here is used for an example for the service account. 
The policy for the cluster Autoscaler is given in this document from line 54 onwards.